net/redis/Makefile

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (C) 2007-2015 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=redis
+PKG_VERSION:=3.2.6
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://download.redis.io/releases/
+PKG_MD5SUM:=d0e81d1e19f673fd84d01784bf9fb5f0
+
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+MAKE_FLAGS+= \
+	MALLOC="libc" \
+	PREFIX="$(PKG_INSTALL_DIR)/usr"
+
+TARGET_LDFLAGS += -L$(STAGING_DIR)/usr/lib/
```

Comment:
I guess this flag is added to TARGET_LDFLAGS by default and if this is the case this line can be dropped